.travis.yml

```diff
@@ -11,4 +11,30 @@ matrix:
 
 cache: cargo
 
-sudo: false
+sudo: true
+
+services:
+- docker
+
+install:
+- gem install fpm --no-rdoc --no-ri
+- gem install package_cloud --no-ri --no-rdoc
+
+after_success:
+- mkdir opt
+- cp target/debug/weldr opt
+# Use the same docker image as much as possible
+- docker run -v "$(pwd):/src/" cdrx/fpm-centos:7 -s dir -t deb -v 0.1.0 -n weldr -C /src  opt/weldr
```

Comment:
>> $TRAVIS_TAG

Oh, I missed that, actually I was using `$(cargo pkgid| awk -F'#' {'print $2'})` in my sample branch.
>> if [ -n "$TRAVIS_TAG" ];

yes, this makes more sense
>> x86_64-unknown-linux-musl 

yes, looks like [(ref)](https://github.com/rust-lang/rust/issues/31322).
Should I use the `gnu` for now?